1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,42 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+let bodyEl = document.querySelector('body');
+
+// let quotationMark = document.createElement('span')
+// quotationMark.innerHTML = '&#8221';
+
+let quoteParagraphEl = document.createElement('p')
+let randomQuote=pickFromArray(quotes)
+quoteParagraphEl.innerHTML= '&#8221'+randomQuote.quote 
+quoteParagraphEl.className='p'
+
+
+let quoteAuthor = document.createElement('p')
+quoteAuthor.innerHTML= '-' + randomQuote.author
+quoteAuthor.className='author'
+quoteParagraphEl.append(quoteAuthor)
+
+//quoteParagraphEl.append(quotationMark)
+bodyEl.appendChild(quoteParagraphEl)
+
+
+
+
+let button = document.createElement('button')
+button.innerHTML= 'New Quote'
+bodyEl.appendChild(button)
+button.className='button'
+
+button.addEventListener('click', () => {
+  let newQuote = pickFromArray(quotes)
+  quoteParagraphEl.innerHTML= '&#8221'+newQuote.quote
+  quoteAuthor.innerHTML= '-' + newQuote.author
+  quoteParagraphEl.append(quoteAuthor)
```

Comment:
here i'm guessing you are combining colours to set as a background... really interesting🙂